h/static/scripts/annotator/main.js

```diff
@@ -1,9 +1,11 @@
+require('../polyfills');
+
 var extend = require('extend');
 var Annotator = require('annotator');
 
 // Polyfills
 var g = Annotator.Util.getGlobal();
-if (g.wgxpath) {
+if (g.wgxpath && !g.document.evaluate) {
```

Comment:
`wgxpath` [already does this check](https://github.com/google/wicked-good-xpath/blob/320441591b69b02b31a2d19bafcfc1b4de19c034/src/wgxpath.js#L237), for what it's worth.